go.sum

```diff
@@ -165,6 +165,49 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
+github.com/dereknola/kubernetes/staging/src/k8s.io/component-base v1.18.20-k3s1/go.mod h1:sk2sgP6I0EQaSU1ADwJe8oYtNzOHI0QlyPlr/3+vcDc=
+github.com/dereknola/kubernetes/staging/src/k8s.io/cri-api v1.18.20-k3s1 h1:wdioniA8KhHLL+srWyLBOcuks7kr04sp5CATUgccK9w=
+github.com/dereknola/kubernetes/staging/src/k8s.io/cri-api v1.18.20-k3s1/go.mod h1:XnplkOLKxeKbT+y5vvjPWc+eONU9byDTVKI9hsqZTR4=
+github.com/dereknola/kubernetes/staging/src/k8s.io/csi-translation-lib v1.18.20-k3s1 h1:YyCQIdvmX6yEsOKU7hmhxIA6snw295jXO+xqZmWFGbY=
+github.com/dereknola/kubernetes/staging/src/k8s.io/csi-translation-lib v1.18.20-k3s1/go.mod h1:LQxrz5PyHX52C2QZCG0kSk+ImaVXBFpfVWD5K47VYXY=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-aggregator v1.18.20-k3s1 h1:njLnjJ0D9boZhf/I4jDKK0wJPzNhUposm08bd1SqpJg=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-aggregator v1.18.20-k3s1/go.mod h1:0kouigesaO14L8x1Dpi2q3iEI43Zb+TjIMeNddpcvT8=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-controller-manager v1.18.20-k3s1 h1:IU2KOzRPEdgFUPq/E9VZ84bGxuGNfEwgVebVAhRPUQo=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-controller-manager v1.18.20-k3s1/go.mod h1:PT4l/+Eiddqn5FnxLR+19Y6tbTsY0Jm2YyQyBMz+Uks=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-proxy v1.18.20-k3s1 h1:MMs2SoblOYygB+UGCFZux0wNPnlsBmsyAPeupn/Pnuk=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-proxy v1.18.20-k3s1/go.mod h1:rQfCml0+V6Sm8UX+2Ol6aEp5Vx5Ia3jZ80DC4u7pjcc=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-scheduler v1.18.20-k3s1 h1:lfLbheYxBA/lYNMwARFoqha5gXv2ii5vNWvupaY5yJw=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kube-scheduler v1.18.20-k3s1/go.mod h1:x0AgqxB6jm40GkP+Qn3VQGqZq6SEs7AaG52z4H78Lp0=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kubectl v1.18.20-k3s1 h1:boIWEG/gQXthZ747GqRw/oOyPHyuW4uaE0Qw18zXpag=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kubectl v1.18.20-k3s1/go.mod h1:EVnvFvv90NpwDnScQlb4sI7rv68MOioOPBYAEjGEwXk=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kubelet v1.18.20-k3s1 h1:ywo6vmOONEFW0G+nU7/FoWmrKeH1B2YR9TGv/3wx1SI=
+github.com/dereknola/kubernetes/staging/src/k8s.io/kubelet v1.18.20-k3s1/go.mod h1:vGzo6okdIPUlm1uKqvArNIb6lXo5aGAT4zXyrlCoMMA=
+github.com/dereknola/kubernetes/staging/src/k8s.io/legacy-cloud-providers v1.18.20-k3s1 h1:f+IVIMC2I6ce7+WtpfOeY8eWcCsA8stMVuPWlpdltcA=
+github.com/dereknola/kubernetes/staging/src/k8s.io/legacy-cloud-providers v1.18.20-k3s1/go.mod h1:BBJvgp6V/akRY2bCFkJw+8xKx9o3sdabCbv1t0okkW4=
+github.com/dereknola/kubernetes/staging/src/k8s.io/metrics v1.18.20-k3s1 h1:6MEVKOjsQTFtC6wU1PngYhyYhlaX/Q+OVjw+qvWIdGQ=
+github.com/dereknola/kubernetes/staging/src/k8s.io/metrics v1.18.20-k3s1/go.mod h1:E7cbaHkNadVcB+b9CqXqoRa++EwOikkSZTdGaJFHm58=
+github.com/dereknola/kubernetes/staging/src/k8s.io/sample-apiserver v1.18.20-k3s1/go.mod h1:lIYmFFDZSe2V4cvuXEVfgXo/9igVESR7VVV6i1kDOyw=
```

Comment:
Needs another `go mod tidy` to get go.sum back in sync now that go.mod is off your personal fork.